low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -180,6 +180,23 @@ SYSTEMDWATCHDOG:
    monitor: true
    threaded: true
    monitored_services:
+      - hare-consul-agent.service
+      - hare-consul-agent-c1.service
+      - hare-consul-agent-c2.service
+      - elasticsearch.service
+      - kibana.service
+      - statsd.service
+      - rsyslog.service
+      - haproxy.service
+      - slapd.service
+      - corosync.service
+      - pacemaker.service
+      - pcsd.service
+      - lnet.service
```

Comment:
Restarting of lnet.service when cortx stack is up is risky, corresponding dependent services needs to be taken care of.

Reply:
Agree, @ajaysrivas @indrajitzagade please make note of this, HA policies should be very careful while touching lnet service

Reply:
Added condition in code to check service status before processing restart/start request, if service state is active then it will not restart/start service again.

Reply:
Yes @ajaysrivas, SSPL will give service failure alerts and take request to start/stop/restart services too and yes we will add check for its active state and start/restart only if not running.